vllm/sequence.py

```diff
@@ -121,6 +124,8 @@ class RequestMetrics:
     scheduler_time: Optional[float] = None
     model_forward_time: Optional[float] = None
     model_execute_time: Optional[float] = None
+    time_per_prefill_token: Optional[float] = None
+    num_evicted_tokens: int = 0
```

Comment:
There is no place in the code where this value is updated. It seems that lines are crossed between V0 and V1 for updating this value.